index.html

```diff
@@ -22,7 +22,7 @@
       </ul>
     </div>
     <header>
-      <div class="logo">My logo</div>
+      <a class="logo">My logo</a>
       <nav>
```

Comment:
- [Optional] Good job making your logo a link, But as you can see the font size is very small and not as visible on the figma design. Please consider updating the font size or wrapping your link inside a `h2` tag, to make it stand out. 

#### Something like. 
```html
<h2><a class="logo">My logo</a></h2>
```
- [Optinal] Also, consider adding some spacing on the top, This Is to align your design better with the figma design. 
![Screenshot 2023-02-15 102257](https://user-images.githubusercontent.com/69077061/219350356-defc1ce9-6e04-45f1-b21d-4380be7f4e49.png)